src/lib.rs

```diff
@@ -68,6 +68,12 @@ pub(crate) struct DebugLinesConfig {
     depth_test: bool,
 }
 
+
+#[derive(SystemSet, Debug, Hash, PartialEq, Eq, Clone)]
+pub enum DebugLinesSet {
+    DrawLines,
```

Comment:
I'm not sure if this enum should be public or not (or if we even need to nest this into it's own set).

Reply:
I think we just leave it public for the sake of simplicity.  If people need to access it then they can.  If they don't, that's fine.